File: data/adventures/app/models/adventure.py
from typing import List, Optional
from app.models.address import Address
from tiklib.models.partner_entity import PartnerEntity

from app.models.ticket import Ticket

class Adventure(PartnerEntity):
    name: Optional[str]
    description: Optional[str]
    tickets: Optional[List[Ticket]]
    venue_id: Optional[str]

File: api/src/models/adventure.ts
import { Address } from './address';

interface Venue {
  name: string;
  address: Address;
  phone: string;
}

interface Ticket {
  capacity: number;
  type: string;
}

export interface Adventure {
  name: string;
  description: string;
  tickets: Array<Ticket>;
  venue: Venue;
}
